frontend/src/FullPageRoutes/Matching.tsx

```diff
@@ -222,7 +279,9 @@ export default function MatchingPage(): JSX.Element {
         "/api/matches/" +
         queryParams({
           ...filters,
-          interests: filters.interests.map((v) => v.id),
+          interests: filters.interests
+            ? filters.interests.map((v) => v.id)
+            : undefined,
```

Comment:
I prefer to avoid undefined whenever possible. What do you think of this? I haven't tested it, but it's the same syntax I used in `fetch.tsx`

```suggestion
          ...(filters.interests && {"interests": filters.interests.map((v) => v.id)})
```

Reply:
Yea I think I like your change better.